src/structures/display/GridDisplay.ts

```diff
@@ -0,0 +1,58 @@
+import { EmbedField } from "eris";
+import { MessageEmbed } from "../client/RichEmbed";
+
+export class GridDisplay {
+  constructor(private items: string[]) {
+    if (items.length > 36) {
+      throw new Error("Can't display more than 36 items!");
+    }
+  }
+
+  public render(embed: MessageEmbed): void {
+    const chunks = this.chunkItems();
+
+    chunks
+      .map((chunk) => ({
+        name: "\u200b",
+        value: chunk.join("\n"),
+        inline: true,
+      }))
+      .forEach((field) => embed.addField(field));
+  }
+
+  public getFields(): EmbedField[] {
+    return this.chunkItems().map((chunk) => ({
+      name: "\u200b",
+      value: chunk.join("\n"),
+      inline: true,
+    }));
+  }
+
+  private chunkItems(): string[][] {
+    if (this.items.length <= 4)
+      return this.createChunks({ into: 4, groupsOf: 1 });
```

Comment:
just wanted to highlight that I could've done 
```ts
return this.createChunks(4, 1)
```
but I used an object to make it clear what those numbers actually mean so you don't just have magic numbers hanging around